### ISSUE TITLE
MultipleInstanceError

### ISSUE THREAD

**Issue by kalmarek** [sympy/sympy#13567]
Latest (`5.0.1`) Anaconda's `sympy-1.1.1` x86-64 with `python-3.6`;
This happens when running **in jupyter notebook** (works fine in standard console)
```
from sympy import *
init_session()
```
I can however run
```
init_printing()
x = Symbol('x')
y = Symbol('y')
x**2 + sqrt(y)
```
just fine.

```
---------------------------------------------------------------------------
MultipleInstanceError                     Traceback (most recent call last)
<ipython-input-7-13181632695b> in <module>()
----> 1 init_session()

~/anaconda3/lib/python3.6/site-packages/sympy/interactive/session.py in init_session(ipython, pretty_print, order, use_unicode, use_latex, quiet, auto_symbols, auto_int_to_Integer, str_printer, pretty_printer, latex_printer, argv)
    431     else:
    432         ip = init_ipython_session(argv=argv, auto_symbols=auto_symbols,
--> 433             auto_int_to_Integer=auto_int_to_Integer)
    434 
    435         if V(IPython.__version__) >= '0.11':

~/anaconda3/lib/python3.6/site-packages/sympy/interactive/session.py in init_ipython_session(argv, auto_symbols, auto_int_to_Integer)
    262         # don't draw IPython banner during initialization:
    263         app.display_banner = False
--> 264         app.initialize(argv)
    265 
    266         if auto_symbols:

<decorator-gen-112> in initialize(self, argv)

~/anaconda3/lib/python3.6/site-packages/traitlets/config/application.py in catch_config_error(method, app, *args, **kwargs)
     85     """
     86     try:
---> 87         return method(app, *args, **kwargs)
     88     except (TraitError, ArgumentError) as e:
     89         app.print_help()

~/anaconda3/lib/python3.6/site-packages/IPython/terminal/ipapp.py in initialize(self, argv)
    315         self.init_path()
    316         # create the shell
--> 317         self.init_shell()
    318         # and draw the banner
    319         self.init_banner()

~/anaconda3/lib/python3.6/site-packages/IPython/terminal/ipapp.py in init_shell(self)
    331         self.shell = self.interactive_shell_class.instance(parent=self,
    332                         profile_dir=self.profile_dir,
--> 333                         ipython_dir=self.ipython_dir, user_ns=self.user_ns)
    334         self.shell.configurables.append(self)
    335 

~/anaconda3/lib/python3.6/site-packages/traitlets/config/configurable.py in instance(cls, *args, **kwargs)
    421             raise MultipleInstanceError(
    422                 'Multiple incompatible subclass instances of '
--> 423                 '%s are being created.' % cls.__name__
    424             )
    425 

MultipleInstanceError: Multiple incompatible subclass instances of TerminalInteractiveShell are being created.
```

**Comment by ylemkimon** [comment]
Duplicate of #13319 and #13418, and a fix is available at #13300.

**Comment by kalmarek** [comment]
yeah, sorry for yet another duplicate; google didn't find the other two issues, but github search does;

closing this
